packages/docusaurus-theme-classic/src/theme/Layout/styles.module.css

```diff
@@ -5,23 +5,21 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-html,
-body {
+:global(html, body) {
   height: 100%;
 }
 
-#__docusaurus {
-  min-height: 100%;
-  display: flex;
-  flex-direction: column;
-}
-
-.main-wrapper {
+.mainWrapper {
   flex: 1 0 auto;
 }
 
-/* Docusaurus-specific utility classes */
-
-.docusaurus-mt-lg {
+/* Docusaurus-specific utility class */
+:global(.docusaurus-mt-lg) {
```

Comment:
not really liking this solitary utility class 🤪 maybe we should just duplicate the rule everywhere 🤷‍♂️